client/src/Molecules/Footer/FooterDesktop.js

```diff
@@ -0,0 +1,86 @@
+import FooterDesktopInnerStyled from './FooterDesktopInnerStyled';
+import FooterDesktopStyled from './FooterDesktopStyled';
+import logoDevsOnTheWaves from '../../Assets/logoDevsOnTheWaves.svg';
+import { Copy, DevsLogo } from './FooterStyled';
+import { LogoStyled } from './FooterStyled';
+
+const FooterDesktop = () => {
+  return (
+    <>
+      <FooterDesktopStyled>
+        <FooterDesktopInnerStyled>
+          <a
```

Comment:
You could probably also use <Link> component from react-router

Reply:
react-router Link does not work for outside links. Link will be used for react routes, a tag will be used for outside links.